src/content/docs/en/guides/images.mdx

```diff
@@ -656,7 +656,7 @@ If the image is merely decorative (i.e. doesn’t contribute to the understandin
 When using a [strict package manager](https://pnpm.io/pnpm-vs-npm#npms-flat-tree) like `pnpm`, you may need to manually install Sharp into your project even though it is an Astro dependency:
 
 ```bash
-pnpm install sharp
+pnpm add sharp
```

Comment:
I'm not completely sure about this one, i'd stick with a fresh re install of sharp tho.

Reply:
No problem! Let's keep the original one